flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -537,7 +543,6 @@ public YarnClusterClient createCluster(
 		Preconditions.checkNotNull(userJarFiles, "User jar files should not be null.");
 
 		AbstractYarnClusterDescriptor yarnClusterDescriptor = createDescriptor(applicationName, cmdLine);
-		yarnClusterDescriptor.setFlinkConfiguration(config);
```

Comment:
This might be the case right now, but it could also change in the future. The signature of the `createCluster` method contains the configuration parameter and, thus, every user will think he can pass a configuration  for the creation. We should keep that way.